bundle/libraries/filer.go

```diff
@@ -24,6 +25,7 @@ func GetFilerForLibraries(ctx context.Context, b *bundle.Bundle) (filer.Filer, s
 	}
 
 	uploadPath := path.Join(b.Config.Workspace.ArtifactPath, InternalDirName)
+	uploadPath = ensureWorkspaceOrVolumesPrefix(uploadPath)
```

Comment:
Could you add a comment why this is needed? Unclear why we would be missing prefix now.

Reply:
Added, it's not that we miss it now, I just moved it from the other place to here, but I added the comment to the function